### DISCUSSION TITLE
i can't create .github repository on my organizations account

### DISCUSSION THREAD

**Discussion by mhmyu** [community/community#47704] [Repositories]
### Select Topic Area

Bug

### Body

i got this The repository .github has been retired and cannot be reused. when i create .github repository for my org accout nextism.
can you help solve this problem.
![Screenshot from 2023-02-17 06-41-54](https://user-images.githubusercontent.com/6158924/219503507-8310c5b3-5940-462d-92c4-5c874a16756f.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mhmmyu ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
